xdsl/dialects/riscv.py

```diff
@@ -439,6 +439,7 @@ class CsrReadWriteImmOperation(IRDLOperation, ABC):
     def __init__(
```

Comment:
```suggestion
    immediate: OptOpAttr[AnyIntegerAttr]

    def __init__(
```
Please add the immediate attribute on the type also